### PR TITLE
Remove references to pre-canton sandbox from documentation

### PR DESCRIPTION
**Pull request by mziolekda**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/ops/connect/auth0.rst [hunk 1]
```diff
@@ -695,7 +695,7 @@ production ledger (minus persistence).
     daml sandbox --ledgerid %%LEDGER_ID%% \
                  --auth-jwt-rs256-jwks https://%%AUTH0_DOMAIN%%/.well-known/jwks.json \
                  --implicit-party-allocation false \
```
Comment (cocreature) [inline docs/source/ops/connect/auth0.rst:697]: this is still wrong, the other args don’t work either but not more wrong than before :shrug: Not for this PR to fix

File: docs/source/support/component-statuses.rst [hunk 2]
```diff
@@ -203,18 +203,6 @@ Developer Tools
...
-   * - Daml Sandbox in Memory (ie without the ``--sql-backend-jdbcurl`` flag)
-     - Stable
-     -
-   * - Daml Sandbox on Postgres (ie with the ``--sql-backend-jdbcurl`` flag)
-     - Stable, Deprecated
-     - 2020-12-16
-   * - Daml Sandbox Classic and associated CLIs ``daml sandbox-classic``, ``daml start --sandbox-classic``
-     - Stable, Deprecated
-     - 2020-04-09
-   * - Daml Profiler ``daml sandbox --profile-dir``
```
Comment (cocreature) [inline docs/source/support/component-statuses.rst:215]: I don’t think we should drop this. The profiler still works and exists, it’s just activated differently. I’d drop the command line and myabe rephrase it to `Daml Profiler in Sandbox`.

File: docs/source/ops/connect/auth0.rst (hunk 1 above)
Comment (mziolekda) [reply docs/source/ops/connect/auth0.rst:697]: Indeed, fixing it takes a bit more than just changing the command line: the equivalent path for canton sandbox needs to be described. It is easier to do it when the old text is still present for reference.

File: docs/source/support/component-statuses.rst (hunk 2 above)
Comment (mziolekda) [reply docs/source/support/component-statuses.rst:215]: done

**Review by cocreature — APPROVED** [review]
Thanks a lot for taking care of this!
